scripts/snapshot-release.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# stop at first command failure
+set -e
+
+if [ "$TRAVIS" != "true" ]; then
+    echo "This script is supposed to be run inside the travis environment."
+    exit 1
+fi
+
+#if [ $TRAVIS_PULL_REQUEST != "false" ]; then
+#    # Dont build anything for PR requests, only for merges
+#    exit 1
+#fi
+
+#if [ $TRAVIS_BRANCH != "master" ]; then
+#    # only update when the target branch is master
+#    exit 1
+#fi
```

Comment:
Only for testing as well, otherwise the script would not be executed regarding to this PR.